tools/missing-test-detector/detector_test.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"reflect"
+	"sort"
+	"testing"
+)
+
+func TestDetectMissingTests(t *testing.T) {
```

Comment:
It looks like you need to add a couple tests here:

- config_variable resource
- a fully uncovered resource

Reply:
Done.